charts/redis/templates/redis-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.managed_redis.disabled }}
```

Comment:
This if statement is not required here. If we want a managed DB, we won't even deploy this entire chart. We can remove it.

Reply:
Yup, It's updated